usr/lib/qubes-whonix/replace-ips

```diff
@@ -39,8 +40,11 @@ import re
 import subprocess
 
 WHONIX_IP_GATEWAY = '/var/cache/qubes-whonix/whonix-ip-gateway'
+WHONIX_IP6_GATEWAY = '/var/cache/qubes-whonix/whonix-ip6-gateway'
 WHONIX_IP_LOCAL = '/var/cache/qubes-whonix/whonix-ip-local'
+WHONIX_IP6_LOCAL = '/var/cache/qubes-whonix/whonix-ip6-local'
 IP_ADDRESS = re.compile(r'(?:[0-9]{1,3}\.){3}[0-9]{1,3}')
+IP6_ADDRESS = re.compile(r'(?:[0-9a-f]{0,4}:){2,7}[0-9a-f]{1,4}', re.IGNORECASE)
```

Comment:
This is probably just a case of me not quite understanding what might be thrown at this regex, but it looks like it will always require the last hextet of the address to be explicitly defined. Elsewhere in your PRs though there are addresses that have the last hextet omitted, for instance `fd09:24ef:4179::a8a:/112`. Will this regex ever have to handle an address like this, or is this not designed to match addresses like that?

Reply:
Upon closer inspection, it looks like this probably will only have to deal with IPv6 addresses with the last hextet defined, so this is likely fine.